dask/dataframe/groupby.py

```diff
@@ -1023,6 +1023,15 @@ def _cumcount_aggregate(a, b, fill_value=None):
     return a.add(b, fill_value=fill_value) + 1
 
 
+def _fillna_groups(groups, axis, **kwargs):
```

Comment:
Nit: this is applying to a single group, so I think `group` would be a more appropriate name

Reply:
Rather than a `**kwargs`, I'd just include the rest of the args explicitly

Reply:
Sounds good to me, shall do!